src/main/java/com/coveo/nashorn_modules/Module.java

```diff
@@ -71,9 +78,23 @@ public Object require(String module) throws ScriptException {
 
     Module found = null;
 
+    Folder resolvedFolder = resolveFolder(folder, folderParts);
+
+    String requestedFullPath = null;
+    if(resolvedFolder != null){
```

Comment:
If you run a compile using Maven the reformatter should add spaces here and at a couple of other places...